app/views/deploys/new.html.erb

```diff
@@ -24,6 +24,17 @@
 
       <%= Samson::Hooks.render_views(:deploy_form, self, project: @project, form: form) %>
 
+      <% if @recent_releases %>
```

Comment:
Ah okay, I don't know why that didn't seem to work earlier. I'll revert the commit and see if CI passes again!